.github/workflows/ci.yml

```diff
@@ -17,9 +17,10 @@ jobs:
       with:
         fetch-depth: 0
     - uses: coursier/cache-action@v5
-    - uses: olafurpg/setup-scala@v10
+    - uses: actions/setup-java@v3
       with:
-        java-version: adopt@1.11
+        java-version: 11
+        distribution: adopt
```

Comment:
Do you mind also updating the caching setup to rely on `setup-java` instead of `cache-action`?

I think removing `- uses: coursier/cache-action@v5` (line 19) and adding `cache: 'sbt'` here would do it.